README.md

```diff
@@ -6,6 +7,8 @@
 This template implements a typical worklist floorplan, one of the design patterns that is specified by the [SAP Fiori Design Guidelines](https://experience.sap.com/fiori-design/). 
 It includes generic application functionality and tests that can be easily extended. You can find more documentation on this template [here](https://openui5.hana.ondemand.com/#/topic/a77f2d29299247f8a3e30226507b1765).
 
+You can also see the [template](http://sap.github.io/openui5-worklist-app/test/mockServer.html) live.
```

Comment:
Suggestion to simplify the text:

This template implements a typical worklist floorplan, one of the design patterns that is specified by the SAP Fiori Design Guidelines. It includes generic application functionality and tests that can be easily extended.

## More information
* [Demo](link)
* [Documentation](link)
* [Design Guidelines](link)